pinax/messages/tests/tests.py

```diff
@@ -233,6 +233,72 @@ def test_unread(self):
             "READ",
         )
 
+    def test_unread_threads_one_unread(self):
+        """
+        Ensure `unread_threads` template_tag produces correct results for one unread message
+        """
+        thread = Message.new_message(
+            self.brosner,
+            [self.jtauber],
+            "Why did you break the internet?", "I demand to know.").thread
+
+        tmpl = """
+               {% load pinax_messages_tags %}
+               {% if user|unread_threads %}{{ user|unread_threads }}{% endif %}
+               """
+        self.assert_renders(
+            tmpl,
+            Context({"thread": thread, "user": self.jtauber}),
```

Comment:
Context does not need "thread" variable. Simplify and clarify by removing "thread" in this and other tests.